### DISCUSSION TITLE
Installation Guide

### DISCUSSION THREAD

**Discussion by hyperdevbootcamps** [elevate/elevate.metabase.tools#33] [Q&A]
Hey Folks, 

Does anyone have an installation guide on how to set up and execute this tool?

**Comment by mausch** [comment]
I just switched to Nix Flakes so you can now use Nix or Docker (docker images are somewhat old by now though)
https://github.com/elevate/elevate.metabase.tools?tab=readme-ov-file#elevatemetabasetools
